source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Controllers/EvaluationsController.cs

```diff
@@ -121,6 +121,7 @@ public ActionResult PeriodEvaluation(string period)
             var isEvaluated = evaluation.Any(p => p.UserName == loggedUser);
             ViewBag.Period = period;
             ViewBag.hasAutoCalification = isEvaluated;
+            ViewBag.isEvaluationRedyForDevolution = evaluation.Any(e => e.UserName == loggedUser && e.OpenToDevolution);
```

Comment:
since we want to know the logged employee's evaluation state, I'll change the name to `isAutoEvaluationOpenForDevolution`